src/content/tools/mcp-server.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: Dart MCP server
+title: Dart and Flutter MCP server
 short-title: MCP server
 description: >-
-  Learn about the Dart MCP server tool that
+  Learn about the Dart and Flutter MCP server tool that
   exposes Dart and Flutter tools to compatible
   AI-assistant clients and agents.
 ---
 
-The Dart MCP server exposes Dart (and Flutter)
```

Comment:
Unrelated to this PR, but do consider adding a cross link to the MCP site/docs for those not familiar.

https://modelcontextprotocol.io/

Reply:
Made a slightly different change to explain what MCP is.